.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+# Needed to enable tracing support for tokio
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
my understanding of this flag is that it only enables opt-in features that are subject to change without corresponding semver changes